test/test_rosmaster/test/client_verification/test_slave_api.py

```diff
@@ -124,7 +124,7 @@ def setUp(self):
                 time.sleep(0.1)
         if not self.node_api:
             self.fail("master did not return XML-RPC API for [%s, %s]"%(self.caller_id, self.test_node))
-        print "[%s] API  = %s"%(self.test_node, self.node_api)
+        print("[%s] API  = %s"%(self.test_node, self.node_api))
```

Comment:
Just noting usually `from __future__ import print_function` would be used, but it looks like there's no difference in output in this case.

Reply:
As long as the code only prints a single string argument you don't need the future import.

When the code passes multiple string arguments you might want the future import in order to not change the behavior change what separator is used between the arguments.

When the code uses `file=sys.stderr` you certainly need the future import to make it work in Python 2.

See http://wiki.ros.org/python_2_and_3_compatible_code#Print_is_now_a_function which describes those cases.